sos/plugins/openstack_cinder.py

```diff
@@ -45,6 +47,9 @@ def setup(self):
         else:
             self.add_copy_spec_limit("/var/log/cinder/*.log",
                                      sizelimit=self.limit)
+        if self.get_option("exclude"):
+            exclude = self.get_option("exclude")
+            self.add_forbidden_path(exclude)
```

Comment:
> It's set on line 51 and used on line 52.

For what possible benefit? If you are going to do that then place the assignment outside of the conditional block so that you don't call `get_option()` twice - right now you are caching something in order to use it exactly once:

```python
        if self.get_option("exclude"):
            exclude = self.get_option("exclude")
            self.add_forbidden_path(exclude)
```